Gemfile

```diff
@@ -10,7 +10,7 @@ gem 'stanford-mods', '~> 2.6'
 gem 'lockfile'       # file locks needed for mutual exclusion during rollup and index addition processes
 gem 'mini_exiftool'  # was-seed-preassembly thumbnail creation
 gem 'mini_magick'    # was-seed-preassembly thumbnail creation
```

Comment:
Can this line be removed now?

Reply:
It looks like minimagick is still used https://github.com/sul-dlss/was_robot_suite/blob/b047c1d9743e330b468c3ce06dd151a8f1014a04/lib/dor/was_seed/thumbnail_generator_service.rb#L35  Should this get replaced with Vips?

Reply:
I saw it used somewhere, I'll double check.

Reply:
I'm going to address this in a separate PR;  I made issue #497 about this and other problem children, as well as leaving #458 open until I get rid of mini_magick and maybe even mini_exiftool.